pkg/jsonlog/jsonlog_marshalling.go

```diff
@@ -81,8 +91,6 @@ func (mj *JSONLog) MarshalJSONBuf(buf *bytes.Buffer) error {
 	if len(mj.Log) != 0 {
 		if first == true {
```

Comment:
Oh well that's right, I <del>could</del>should remove this if altogether :sweat:.